### DISCUSSION TITLE
Autocomplete not working for ZSH

### DISCUSSION THREAD

**Discussion by pro100svitlo** [fastlane/fastlane#19647] [Q&A]
there was a feature request [here](https://github.com/fastlane/fastlane/issues/883)

I followed steps:
- run `fastlane enable_auto_complete`
- copied `. ~/.fastlane/completions/completion.sh` to my `~/.zshrc`
- run `source ~/.zshrc`
- tried `fasttlane <TAB>` but nothing happened 😢 

Did I miss some step? Or there is some bug?

**Comment by lucgrabowski** [comment]
Hi, try editing zshrc to use script with `.zsh` extension: `. ~/.fastlane/completions/completion.zsh`. It should help.
